project.php

```diff
@@ -74,10 +88,13 @@ class=\"float-right\">
 
                     $displayTaskName = $task['task_name'];
                     $displayTaskEstimate = $task['estimate'];
+                    $estimate_us = EstimateService::convertEstimate($displayTaskEstimate);
 
+                    if ($pageLocale == 'us') {
+                        $displayTaskEstimate = $estimate_us;
+                    }
```

Comment:
As before, if `$estimate_us` isn't being reused, you don't need the variable assignment.

In fact, you could replace a whole 5 lines with:

```
$displayTaskEstimate = ($pageLocale === 'us') ? EstimateService::convertEstimate($displayTaskEstimate) : $task['estimate'];
```

Reply:
Also, quite inconsistent variable naming here: pick either camelCase or snake_case and stick with just the one you've chosen, not both!